titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/appscale/AppAutoScalingCallbackSpringResource.java

```diff
@@ -56,6 +56,7 @@ public ScalableTargetResourceInfo getScalableTargetResourceInfo(@PathVariable("s
         if (scalableTargetResourceInfo.getDesiredCapacity() < 0) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
         }
-        return Responses.fromSingleValueObservable(awsGatewayCallbackService.setScalableTargetResourceInfo(jobId, scalableTargetResourceInfo, authentication.getCallMetadata()));
+        ScalableTargetResourceInfo updatedScalableTargetResourceInfo = Responses.fromSingleValueObservable(awsGatewayCallbackService.setScalableTargetResourceInfo(jobId, scalableTargetResourceInfo, authentication.getCallMetadata()));
+        return ResponseEntity.status(HttpStatus.OK.value()).body(updatedScalableTargetResourceInfo);
```

Comment:
Could you fix the other method as well?

Reply:
Actually the other method should be fine. We return `ScalableTargetResourceInfo`, not `ResponseEntity<ScalableTargetResourceInfo>`